packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -95,14 +129,21 @@ class DescopeWc extends BaseDescopeWc {
         const { body } = await fetchContent(htmlLocaleUrl, 'text');
         return body;
       } catch (ex) {
-        this.logger.error(
+        this.logger().error(
           `Failed to fetch html from ${htmlLocaleUrl}, error: ${ex}. Fallback to url ${htmlUrl}`
         );
       }
     }
 
-    const { body } = await fetchContent(htmlUrl, 'text');
-    return body;
+    try {
+      const { body } = await fetchContent(htmlUrl, 'text');
+      return body;
+    } catch (ex) {
+      this.logger().error(
```

Comment:
can you explain why this change is needed?

the log addition looks good,
but I think that throwing is better is more suitable here than returning null
it will signal caller that the action failde

Reply:
this is the only place that throw an error. i would expect this error to be part of the sdk as in the logs and same as the other errors to be passed as an error, otherwise we and other users need to get the error details from different places